tests/framework/integration/cluster.go

```diff
@@ -1383,6 +1383,13 @@ func (c *Cluster) ClusterClient() (client *clientv3.Client, err error) {
 			DialTimeout: 5 * time.Second,
 			DialOptions: []grpc.DialOption{grpc.WithBlock()},
 		}
+		if c.Cfg.ClientTLS != nil {
+			tls, err := c.Cfg.ClientTLS.ClientConfig()
+			if err != nil {
+				return nil, err
+			}
+			cfg.TLS = tls
+		}
```

Comment:
nice, it should be reason why previously it did not work. Actually I fixed it in [13690](13690/files).

Reply:
ok